ci/vmtest/run_selftests.sh

```diff
@@ -81,13 +102,22 @@ echo "ALLOWLIST: ${ALLOWLIST}"
 
 cd ${PROJECT_NAME}/selftests/bpf
 
-if [ $# -eq 0 ]; then
+# populate TEST_NAMES_FROM_BOOT
```

Comment:
Not a fan of having a function populate a global variable as a side effect, but I suppose that's just a `bash`-ism we have to live with?

Reply:
yeah, bash does not have many options in that regard. The alternatives are not necessarily less complex.
https://www.linuxjournal.com/content/return-values-bash-functions has some options for returning values, but it does not seem easier to maintain.